### ISSUE TITLE
default behaviour is changing http destination urls to https

### ISSUE THREAD

**Issue by carlosjs23** [ash-jc-allen/short-url#35]
When I set a destinationUrl that is http (not https) its changed to https after building the ShortUrl.

```
$route = route('invoices.print', $factura->id); // http://localhost:8000/invoices/3/print
$shortURLObject = $builder->destinationUrl($route)->make();
$shortURLObject->destionation_url; // https://localhost:8000/invoices/3/print
```

We already have a method to force HTTPS:

> Enforce HTTPS
   When building a shortened URL, you might want to enforce that the visitor is redirected to the HTTPS version of the destination URL. This can be particularly useful if you're allowing your web app users to create their own shortened URLS. 
    To enforce HTTPS, you can use the ->secure() method when building the shortened URL.

So I think the default behaviour is ambiguous with this method, should we change it to respect http urls?

Edit: I've found that using ``->secure(false)`` solves this, but I still think that replacing http by https by default can be unexpected.

**Comment by ash-jc-allen** [comment]
You make a good point on this. If you'd prefer to not make the short URL use HTTPS you can use the following:

```
$shortURLObject = $builder->destinationUrl($route)->secure(false)->make();
```
In the builder class, the secure option defaults to true. However, I do actually have plans to make this into a config variable in a future release. If this is something you'd like to add to it, I'd be more than happy to accept any pull requests. It'd help me out and get the feature added sooner :)

**Comment by ash-jc-allen** [comment]
This is solved in PR #36. It will be added to release v2.2.0
